sycl/include/sycl/detail/pi.h

```diff
@@ -2225,6 +2225,90 @@ __SYCL_EXPORT pi_result piextCommandBufferMemBufferCopyRect(
     const pi_ext_sync_point *sync_point_wait_list,
     pi_ext_sync_point *sync_point);
 
+/// API to append a mem buffer read command to the command-buffer.
+/// \param command_buffer The command-buffer to append onto.
+/// \param src is the data to be read
+/// \param offset offset into \p src
+/// \param size is number of bytes to read
+/// \param dst is the pointer to the destination
+/// \param num_sync_points_in_wait_list The number of sync points in the
+/// provided wait list.
+/// \param sync_point_wait_list A list of sync points that this command must
+/// wait on.
+/// \param sync_point The sync_point associated with this memory operation.
+__SYCL_EXPORT pi_result piextCommandBufferMemBufferRead(
+    pi_ext_command_buffer command_buffer, pi_mem src, size_t offset,
```

Comment:
think the `pi_mem src` parameter should be named `pi_mem buffer` to be consistent.